.gitignore

```diff
@@ -14,3 +14,10 @@ webrev/
 webrev.zip
 gradle.properties
 apps/samples/Ensemble8/lib/
+
+.idea/tasks.xml
+.idea/workspace.xml
+.idea/inspectionProfiles/*
+.idea/copyright/profile_settings.xml
```

Comment:
I believe it should be `profiles_settings.xml` as plural. I also believe this has no reason to exist in the git repository because IntelliJ can auto-generate it. I am not sure exactly what it is for but as an example the one from my local repo is:

```xml
<component name="CopyrightManager">
  <settings default="" />
</component>
```

I will change this PR to exclude `.idea/copyright/profiles_settings.xml`.